testgrid/specs/full.yaml

```diff
@@ -1263,4 +1263,11 @@
       rookShouldUseAllNodes: false
       shouldDisableRebootServices: false
       shouldDisableClearNodes: false
-      shouldEnablePurgeNodes: false
\ No newline at end of file
+      shouldEnablePurgeNodes: false
+  unsupportedOSIDs:
+    - amzn-20
+    - ol-79
+    - ol-85
+    - ubuntu-1604
+    - ubuntu-1804
+    - ubuntu-2004
```

Comment:
This changes are introduced in the main branch already.
So, I am closing this one.